common/lib/dependabot/shared_helpers.rb

```diff
@@ -484,6 +484,64 @@ def self.helper_subprocess_bash_command(command:, stdin_data:, env:)
       env_keys = env ? env.compact.map { |k, v| "#{k}=#{v}" }.join(" ") + " " : ""
       "$ cd #{Dir.pwd} && echo \"#{escaped_stdin_data}\" | #{env_keys}#{command}"
     end
+
+    sig do
```

Comment:
These changes seem out of place in this file. It seems a more appropriate place would be in the PackageManagerBase or in the PullRequestCreator where most of the PR body generation exists.